### PR TITLE
Bugfix: Change password form

### PR DESCRIPTION
**Pull request by BCFCODE**

### CODE REVIEW

**Comment by BCFCODE** [comment]
Hey! I forked you!! can you hear me?

File: README.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 # ForkTraining on Amy's machine
 
 helooooooooooo
+
+this is a commit from my original git account (BCFCODE)
```
Comment (BCFCODE-ColaborationTraining) [inline README.md:6]: After this line write: helooooooooooo

File: README.md [hunk 2]
```diff
@@ -2,3 +2,7 @@
...
+this is another commit
```
Comment (BCFCODE-ColaborationTraining) [inline README.md:7]: this ok, Go ahead.............

Comment (BCFCODE) [reply README.md:7]: Ok! thanks for reviewing!😍

Comment (BCFCODE-ColaborationTraining) [reply README.md:7]: Ok... Add another line here: type Hi BOSS!!

File: README.md (hunk 1 above)
Comment (BCFCODE) [reply README.md:6]: heloooooooooo

**Review by BCFCODE-ColaborationTraining — COMMENTED** [review]
Ok. do it please...

File: README.md [hunk 3]
```diff
@@ -2,3 +2,8 @@
...
+Hi BOSS!!
```
Comment (BCFCODE-ColaborationTraining) [inline README.md:8]: Good job!

**Review by BCFCODE-ColaborationTraining — APPROVED** [review]
Good Job Awesome!

**Pull request merged**